### ISSUE TITLE
Demo website

### ISSUE THREAD

**Issue by brucou** [substance/substance#522]
Don't know if that has already been asked or discussed. But I unsuccessfully tried to install your demos. And I don't really want to dig into package.json and gulpfiles (error was when coming to the point of installing node-sass). Have you ever considered putting your demos on a website, so there is no need for local installation? At least I would be able to evaluate it properly.

**Comment by michael** [comment]
You can find them here:

http://substance.io/examples/

Also: http://substance.io/lens

What was the exact problem with node-sass?

**Comment by brucou** [comment]
Thanks Michael. I removed the whole thing, so I don't have the trace for the log. I will try again later and report it to you. Basically I had to update my version of node at some point as the installation required a newer node version. And after that, it came back with a node-sass error but there were no real information in the error log. I'll try to reproduce it and report. In the meanwhile I am closing this.
